hera_cal/vis_clean.py

```diff
@@ -688,8 +688,10 @@ def fourier_filter(self, filter_centers, filter_half_widths, mode,
                                 flgs[i] = True
 
             filtered_model[k] = mdl
+            filtered_model[k][flgs] = 0.
```

Comment:
so `flgs` here is skipped integrations? Maybe we should change this to `skipped`?

Reply:
yeah, `flgs` are the skipped integrations. I agree with changing the name ;)